src/main.ts

```diff
@@ -48,7 +49,13 @@ function actionHandler(file: Node, dir: string) {
       path: dir,
     });
   } else {
-    downloadUrl = getAbsolutePath(file.encodedSource);
+	// NC28 dependency : Node.encodedSource
+	if (!file.encodedSource) {
+		const { origin } = new URL(file.source)
+		downloadUrl = getAbsolutePath(origin + encodePath(file.source.slice(origin.length)))
```

Comment:
Let's just revert to the old behavior behavior for this case. I don't want to risk breaking some edge cases.

Reply:
As you like. This was keeping the fix you've done in april.